docs/content/en/blog/articles/why-k8ssandra/index.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2021-02-26
+title: "Why K8ssandra?"
+linkTitle: "Why K8ssandra?"
+description: >
+  The inspiration behind the K8ssandra project, and the benefits it brings to developers, operators and SREs.
+author: Jeff Carpenter ([@jscarp](https://twitter.com/jscarp))
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+
+---
+
+## The problem: scalable data persistence for cloud-native applications
+
+The past decade and a half has brought tremendous change in how we in the software industry think about building and delivering internet applications. With the launch of Amazon Web Services (AWS) in 2006, companies of any size could access cloud computing infrastructure. The burst of creativity that followed gave birth to concepts like [cloud-native](http://pzf.fremantle.org/2010/05/cloud-native.html) in 2010, and methodologies such as the [twelve factor app](https://12factor.net/) in 2011.
```

Comment:
I don't think "cloud native" should be hyphenated here; it's not being used as a compound adjective.